packages/pointer_interceptor/pointer_interceptor_web/example/integration_test/widget_test.dart

```diff
@@ -70,7 +71,8 @@ Future<void> _fullyRenderApp(WidgetTester tester) async {
   await tester.pumpWidget(const app.MyApp());
   // Pump 2 frames so the framework injects the platform view into the DOM.
   await tester.pump();
-  await tester.pump();
+  // Give the browser some time to perform DOM operations (for Wasm code)
+  await tester.pump(const Duration(milliseconds: 500));
```

Comment:
Talk to @eyebrowsoffire about this new delay.

Reply:
We discussed this, the skwasm renderer is now async, so it makes sense that there's some timing differences compared to the JS one. Needing this additional delay is probably caused by the `tester` not being aware of this asynchronicity, and declaring a frame "pumped" before it's really been fully rendered.